src/librustc_codegen_ssa/back/write.rs

```diff
@@ -380,6 +323,9 @@ pub fn start_async_codegen<B: ExtraBackendMethods>(
 ) -> OngoingCodegen<B> {
     let (coordinator_send, coordinator_receive) = channel();
     let sess = tcx.sess;
+
+    sess.prof.generic_activity_start("codegen_and_optimize_crate");
```

Comment:
this activity causes the events to get in the faulty order 
i get a panic added som printout to see what event was the problem.

when I removed this trace it worked again

`thread 'main' panicked at 'previous event on thread wasn't the start event , stopEvent Event { event_kind: "GenericActivity", label: "codegen_crate", additional_data: [], timestamp: SystemTime { tv_sec: 117, tv_nsec: 342291500 }, timestamp_kind: End, thread_id: 2 }, Event that do not match as a startEvent Event { event_kind: "GenericActivity", label: "codegen_and_optimize_crate", additional_data: [], timestamp: SystemTime { tv_sec: 62, tv_nsec: 748803800 }, timestamp_kind: Start, thread_id: 2 }', measureme/src/profiling_data.rs:140:`

Reply:
created https://github.com/rust-lang/rust/pull/65137 to remove this event

Reply:
I created https://github.com/rust-lang/rust/pull/65137 to remove the event